spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+  render_views
+  let(:user) { create(:user) }
+
+  before do
+    request.headers['Authorization'] = "Token token=#{user.refresh_token}"
+  end
+
+  describe "#index" do
+    before do
+      5.times{ create(:user) }
+    end
+
+    it "gets basic user information" do
+      get :index, format: :json
+
+      expect(parsed_json).to have_key("users")
+
+      sample_user = parsed_json['users'].first
+      expect(sample_user['id']).to be
```

Comment:
What are you testing here with `expect(sample_user['id']).to be`?
Also it makes sense to group our verification phase together. The 4 phases of testing are:
- Setup
- Execution
- Verification
- Teardown
In this order.
Check out [this blog post](http://www.codeaffine.com/2014/08/18/junit-in-a-nutshell-test-structure/)  and [this blog post](https://robots.thoughtbot.com/four-phase-test)for a better explanation of 4 phase test